### PR TITLE
pythonPackages.sphinx-argparse: init at 0.2.5

### PR DESCRIPTION
**Pull request by clacke**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/sphinx-argparse/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+, buildPythonPackage
+, fetchPypi
+, CommonMark
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-argparse";
+  version = "0.2.5";
+  src = fetchPypi {
```
Comment (dotlambda) [inline pkgs/development/python-modules/sphinx-argparse/default.nix]:
```suggestion

  src = fetchPypi {
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.sphinx-argparse python3.pkgs.sphinx-argparse

File: pkgs/development/python-modules/sphinx-argparse/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    inherit pname version;
+    sha256 = "05wc8f5hb3jsg2vh2jf7jsyan8d4i09ifrz2c8fp6f7x1zw9iav0";
+  };
+
+  propagatedBuildInputs = [
+    CommonMark
```
Comment (dotlambda) [inline pkgs/development/python-modules/sphinx-argparse/default.nix]: As this is an optional dependency, what's your argument for including it by default?

**Comment by dotlambda** [comment]
> Ran 0 tests in 0.000s

Please add an appropriate `checkPhase` or - if there are no tests - set `doCheck = false` and add a comment with the reason.

File: pkgs/development/python-modules/sphinx-argparse/default.nix (hunk 2 above)
Comment (clacke) [reply pkgs/development/python-modules/sphinx-argparse/default.nix]: None. Removed.

**Comment by dotlambda** [comment]
@clacke Please squash the new commits into the initial one.

@GrahamcOfBorg build python2.pkgs.sphinx-argparse python3.pkgs.sphinx-argparse

**Comment by clacke** [comment]
Otherwise good?

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.sphinx-argparse python3.pkgs.sphinx-argparse

**Comment by dotlambda** [comment]
We usually use commit messages formatted like this PR's title, but it's okay as it is.

**Pull request merged**